### ISSUE TITLE
App.js issue again

### ISSUE THREAD

**Issue by Sophrinix** [CodexLabs/TiCalabash#2]
Apparently Appc is back to the App.js issue again.

**Comment by christianjgreen** [comment]
Has there been any updates with this? Keeping getting the app.js issue only when using calabash or building straight from xCode.

**Comment by dfharmon** [comment]
I am also curious if anyone has found a solution or work around for this.   Android works fine for me, but when I use the latest Xcode and calabash-ios to work with iOS 9.3+ the calabash *-cal build always throws the "can't find app.js" issue when launching the iOS app as part of the it calabash command.  When launching directly using ti build -p ios the application runs fine.

**Comment by Sophrinix** [comment]
Yes. I wrote an entire plugin to solve this problem. I still have two remaining issues that keep me from integrating it into this command. https://github.com/sophrinix/fixappjs
